reo/nested_outputs.py

```diff
@@ -164,7 +174,21 @@ def list_of_string(input):
                   "type": float,
                   "description": "Net replacement costs for all technologies, in future value, excluding incentives.",
                   "units": "$"
-                }
+                },
+                "simple_payback_years": {
+                  "type": float,
+                  "description": ("Number of years until the cumulative cashflow is positive. If there are charges "
```

Comment:
I had to read this 3 or 4 times to get the meaning. Can you please re-word it to make it more clear? Maybe something like "Number of years until the cumulative annual cashflow turns positive. If the cashflow becomes negative again after becoming positive then simple payback is increased by the number of years that the cash flow is negative beyond the break-even year."  And now that I've written that, could it be defined as "Number of years that the cumulative annual cashflow is negative, including negative years beyond the break-even year if they exist (for example when paying for storage replacement costs)."